platformio.ini

```diff
@@ -10,23 +10,25 @@
 
 [common]
 board_f_cpu = 160000000L
-lib_deps_builtin = 
+lib_deps_builtin =
   SPI
-lib_deps_external = 
+lib_deps_external =
   RF24
   WiFiManager
   ArduinoJson
-build_flags = !echo -D MILIGHT_HUB_VERSION=\\\"$(git describe --always)\\\"
+build_flags = !echo -D MILIGHT_HUB_VERSION="\"$(git describe --always)\""
```

Comment:
Was the extra string escaping breaking for you?  What platform are you on?  It seemed necessary to double-escape on both OS X and Linux (Travis).